travis-ci/run-standard-tests.sh

```diff
@@ -2,28 +2,22 @@
 set -e
 
 display_and_run() {
-  echo "***" "$@"
-  "$@"
+    echo "***" "$@"
+	eval $(printf '%q ' "$@")
 }
 
+# reset workdir to state from git (to remove possible rewritten dependencies)
+display_and_run git reset --hard
```

Comment:
This won't reset rewritten dependencies in the gx cache. I am not sure what the best solution for that is though.

Reply:
This script is meant to handle repos published in a non rewritten state.
A common pattern in our `.travis` files is to `gx-go rw`. This reverts this operation.